MAUI/ComboBox/Selection.md

```diff
@@ -64,18 +66,81 @@ The following gif image illustrates the result of the above code:
 
 ![.NET MAUI ComboBox programmatic selection](Images/Selection/ProgramaticSelection.png)
 
+## Multiple selection
+
+The ComboBox allows user to select multiple values from the drop-down list. Selected items will be displayed with a customizable token representation and the users can remove each tokenized item by clicking the close button. 
+
+Token mode supports both editable and non-editable text boxes for selecting items from a data source. The multi-select ComboBox mode can be enabled by setting the SelectionMode property as Multiple.
+
+### Programmatic selection
+
+The selected items can be changed programmatically by using the SelectedItems property of ComboBox control.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <editors:SfComboBox x:Name="comboBox" 
+             HeightRequest="50"
+             WidthRequest="350"
+             ItemsSource="{Binding SocialMedias}"
+             SelectionMode="Multiple"
+             MaxDropDownHeight="250"
+             DisplayMemberPath="Name"
+             TextMemberPath="Name" />
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+  SocialMediaViewModel socialMediaViewModel = (this.comboBox.BindingContext as SocialMediaViewModel);
+  ObservableCollection<SocialMedia> socialMediasList = socialMediaViewModel.SocialMedias;
+  this.comboBox.SelectedItems.Add(socialMediasList[0]);
+  this.comboBox.SelectedItems.Add(socialMediasList[2]);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+The following image illustrates the result of the above code.
+
+![.NET MAUI ComboBox Multiple selection](Images/Selection/MultipleSelection.png)
+
+### UI selection
+
+The selected items can be changed interactively by using keyboard or by selecting from a drop-down list. The selected items can be retrieved from the SelectedItems property of ComboBox control.
```

Comment:
The selected items can be changed interactively using a keyboard or selecting from a drop-down list. The selected items can be retrieved from the SelectedItems property of ComboBox control.